README.md

```diff
@@ -608,6 +610,40 @@ The DLP templates used in this tutorial are specifically tailored for inspecting
 2. [Create de-identification templates and run de-identification on sample data](https://cloud.google.com/dlp/docs/creating-templates-deid)
 
 
+## Run this pipeline outside Google Cloud shell
+
+Google Cloud Shell comes with a pre-configured development environment that includes many of the most popular tools and
+libraries, such as gcloud, gradle, java, git, vim, and more. This means that you don't have to spend time setting up 
+your own environment on your local machine. If you want to run this solution from your own machine, follow the 
+instructions given below.
+
+1. Create a new GCP project.
+2. Set up the tools and libraries required to run the DLP Dataflow pipelines including gradle (recommended version: 8.3),
```

Comment:
nit: just use `1.` and allow markdown render correct numbers.

Reply:
It would work fine generally but when there a command element after a list element, it will restart numbering from 1. again.